charts/rancher-vsphere-csi/questions.yaml

```diff
@@ -97,6 +97,13 @@ questions:
     default: false
     group: Storage
 
+  - variable: csiController.csiSnapshotter.enabled
+    label: Enable CSI Volume Snapshotter
+    description: This feature is available for vSphere 7.0 U1+ only
+    type: boolean
+    default: false
+    group: Storage
+
```

Comment:
As we talked offline, we need to warn the user they need to run the setup script first and point out all the requirement and relevant information upstream has stated here:
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/c3be0f6739bd8e5c451ce8eaec6fca538faaf243/manifests/vanilla/deploy-csi-snapshot-components.sh#L20-L38

Reply:
@PennyScissors @brandond I have removed it for now. I think we can get all of this added into the chart to handle it and we can just set it as disabled by default.